img_proof/ipa_ec2.py

```diff
@@ -233,6 +233,14 @@ def _launch_instance(self):
             kwargs['AdditionalInfo'] = self.additional_info
 
         cpu_options = self.custom_args.get('cpu_options', {})
+        for option in self.instance_options:
+            try:
+                key, value = option.split('=')
```

Comment:
For now I'm using the same value as before for SEV. However, we can make this more generic in the future such that we can capture other values aside from CPUOptions. What it is now `AmdSevSnp=enabled`. We could make this `CpuOptions=AmdSevSnp.enabled`. The latter would allows img-proof to generically process values for ec2 such that they end up as key/val. e.g.

```
CpuOptoins={
    'AmdSevSnp': 'enabled'
}
```

Reply:
If we want to avoid a breaking change later the new instance_options value could be switched to `CpuOptions=AmdSevSnp.enabled`.

Reply:
`CpuOptions=AmdSevSnp.enabled`  id documented so the implementation should match. And we do want to, as much as possible, avoid breaking changes later.

Reply:
Yep, I implemented the new argument in a way that handles `CpuOptions=AmdSevSnp.enabled` as preferred. It also will handle the other version `AmdSevSnp=enabled` but that is not documented. So we can drop that code with next major release.